config/job.yaml

```diff
@@ -0,0 +1,55 @@
+# Job Configuration
+
```

Comment:
I think we should add some comments here. things like:
- "an aztk spark job is a cluster an array of spark applications to run on that cluster ..."
- "an aztk spark job will automatically create/delete your cluster"